### ISSUE TITLE
Error trying to embed Spotify playlists URIs

### ISSUE THREAD

**Issue by flrnprz** [thepracticaldev/dev.to#2739]
**Describe the bug**
Trying to embed a Spotify playlist URI gives and error

**To Reproduce**
Example code:
```
{% spotify spotify:user:spotify:playlist:37i9dQZF1E36t2Deh8frhL %}
```

1. Go to 'Write post'
2. Insert the example code
3. Click 'save post' or 'Preview'

**Expected behavior**
Embed the URI the same way with albums, tracks, etc.

**Screenshots**
![screen capture](https://user-images.githubusercontent.com/659650/57293659-a6c04f00-70c5-11e9-8b63-49231189d173.png)

**Additional context**
https://github.com/thepracticaldev/dev.to/issues/1546#issuecomment-490039164

**Comment by Glennmen** [comment]
I played around with the regex and came up with this: https://regex101.com/r/SrZGeP/2
Not sure about the user ID part, I feel like it could be better.

**Comment by Glennmen** [comment]
I combined it with the already available regex: https://regex101.com/r/SrZGeP/3
Don't know if `show` is a valid Spotify URI, it isn't in the current regex.
Also I am not a regex "expert", just playing around with it.

**Comment by flrnprz** [comment]
`Show` is like `album` for podcasts.

**Comment by mariocsee** [comment]
If anyone's opening a PR for this, just a heads up there's a slight refactor to the `render` method in the Spotify Liquid tag, but RegEx parsing methods are untouched.

**Comment by Glennmen** [comment]
If my regex is good for you guys (with `show`) I can make a PR tomorrow.

**Comment by kyleboe** [comment]
Let me know if I can be of any help! I wrote the spotify liquid tag feature so if you have any questions about my reasoning behind things, feel free to ask. 😃

**Comment by maestromac** [comment]
Now that `SpotifyTag#render` is using ERB template, the regex can be loosen bit. Please still watch out for any XSS vulnerbility.

**Comment by Glennmen** [comment]
@flrnprz & @kyleboe I made a PR if you would like to check before it gets merged #2791

**Comment by kyleboe** [comment]
@Glennmen I know I'm late to the game but awesome work on the PR! Love your approach.
